src/modules/events/components/eventCard/index.js

```diff
@@ -1,7 +1,69 @@
-import React, { Component } from "react";
+import React, { Component }  from 'react'
+import {Link} from 'react-router-dom'
+import styles from "./style.module.css";
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+import
+ {   faMapMarkerAlt,
+     faCalendarCheck ,
+     faClock ,
+     faCalendarTimes
+  } from "@fortawesome/free-solid-svg-icons";
 
-export default class EventCard extends Component {
-  render() {
-    return <div></div>;
-  }
+class EventCard extends Component{ 
+
+  render(){
+  
+    const {event} = this.props ;
+     return (
+           <section className={styles["event-card_content"]}>
+              <figure className={styles["events-card_cover"]} key={event}>
+                  <Link to={'/' + event.id}>
+                      <img  
+                        alt={`event ${event.title}`}
+                        src={event.cover}
+                        key={event.id} 
+                        />
+                  </Link>
+                  <p className={styles['activ-upcoming_event']}>
+                    {event.status === "upcoming"?<div>
+                     <FontAwesomeIcon icon={faClock}
+                        className={styles["activ-upcoming-event_icon"]}/></div>: ''}
+                  </p>
```

Comment:
change the p tag to figure for example and the div tag to span to solve console error.